unfetter-discover/unfetter-store
# Custom attack pattern ingest

ingest custom attack patterns

## To test

```
  cd unfetter-store/unfetter-ctf-ingest
  npm install
  npm run build && npm run lint && npm run test
  npm run ingest:attack-pattern -- -f test-data/attack-patterns/attack-patterns-sample.csv
```
The run should display new attack patterns ids, you can find them in the mongo in the stix collection
